src/main/java/net/dries007/tfc/api/capability/heat/CapabilityItemHeat.java

```diff
@@ -39,6 +40,7 @@ public static void preInit()
         CapabilityManager.INSTANCE.register(IItemHeat.class, new DumbStorage<>(), ItemHeatHandler::new);
         //register heat on vanilla egg for cooking
         CapabilityItemHeat.CUSTOM_ITEMS.put(IIngredient.of(Items.EGG), () -> new ItemHeatHandler(null, 1, 480));
+        CapabilityItemHeat.CUSTOM_ITEMS.put(IIngredient.of(ItemsTFC.UNFIRED_BRICK), () -> new ItemHeatHandler(null, 1, 1601));
```

Comment:
I don't believe this is used anymore since the unfired brick uses `ItemPottery`?